openslides_backend/services/media/adapter.py

```diff
@@ -0,0 +1,29 @@
+from typing import Dict, Optional
+
+import requests
+
+from ...shared.exceptions import ActionException
+from ...shared.interfaces.logging import LoggingModule
+from .interface import MediaService
+
+
+class MediaServiceAdapter(MediaService):
+    """
+    Adapter to connect to media service.
+    """
+
+    def __init__(self, media_url: str, logging: LoggingModule) -> None:
+        self.logger = logging.getLogger(__name__)
+        self.media_url = media_url
+
+    def upload(self, file: str, id: int, mimetype: str) -> Optional[Dict[str, str]]:
+        payload = {"file": file, "id": id, "mimetype": mimetype}
+        self.logger.debug(f"Starting upload of file '{file}'")
+        try:
+            response = requests.post(self.media_url, json=payload)
+        except Exception:
```

Comment:
Can you please use explicit excetions here? See https://github.com/OpenSlides/openslides-backend/blob/master/openslides_backend/services/datastore/http_engine.py#L49

Reply:
Done